### PR TITLE
added new approved Build and Deploy verbs

### PR DESCRIPTION
**Pull request by SteveL-MSFT**
Fix https://github.com/PowerShell/PowerShell/issues/3346

removed suggestions that these were not valid

organized verbs to be in alphabetical order to make it easier to compare against
(unfortunately, the diff viewer on GitHub isn't so smart about lines that were moved, easier to review the individual commits)
https://msdn.microsoft.com/en-us/library/ms714428(VS.85).aspx

Created doc bug to update above link https://github.com/PowerShell/PowerShell-Docs/issues/1630

### CODE REVIEW

File: src/System.Management.Automation/utils/Verbs.cs [hunk 1]
```diff
@@ -348,6 +348,11 @@ public static class VerbsLifecycle
         public const string Assert = "Assert";
 
         /// <summary>
+        /// Creates an artifact (usually a binary or ) out of some set of input files (usually source code or declarative documents)
```
Comment (markekraus) [inline src/System.Management.Automation/utils/Verbs.cs]: `(usually a binary or )` Missing info after 'or'.

**Review by markekraus — COMMENTED** [review]
Missing some info

File: src/System.Management.Automation/utils/Verbs.cs (hunk 1 above)
Comment (SteveL-MSFT) [reply src/System.Management.Automation/utils/Verbs.cs]: will fix

**Comment by SteveL-MSFT** [comment]
@markekraus can you `Approve` if you're ok with the changes?  Thanks!

**Review by markekraus — APPROVED** [review]

**Review by daxian-dbw — APPROVED** [review]

**Pull request merged**